objects/src/testing/storage.rs

```diff
@@ -56,6 +43,40 @@ impl AccountStorageBuilder {
         }
         self
     }
+    /// Configures storage slots for a wallet account with authentication.
+    pub fn with_wallet_storage(&mut self, public_key: PublicKey) -> &mut Self {
+        self.add_slot(StorageSlot::Value(public_key.into()));
+
+        self
+    }
```

Comment:
The entire `AccountStorageBuilder` is now unused. Should we remove it?
I think even in cases where we want to build custom storage for tests we should probably construct it with `AccountComponent`s and if that is insufficient modify it on `AccountStorage` directly like you did when setting total issuance in a faucet's reserved slot - what do you think?

Reply:
Yeah, I thought of completely removing this but was not sure anything else from the outside was currently relying on this. I'll remove it.